style.css

```diff
@@ -0,0 +1,172 @@
+@import url('https://fonts.googleapis.com/css2?family=Rock+Salt&display=swap');
+
+* {
+  margin: auto;
+  box-sizing: border-box;
+  font-family: 'Rock Salt', cursive;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  background-color: green;
+  font-family: 'Rock Salt', cursive;
+  text-align: center;
+  overflow-x: hidden;
+}
+
+header {
+  display: flex;
+  position: absolute;
```

Comment:
Why is this absolutely positioned? Seems like disabling flex on the body and removing absolute here has the desired effect